hadoop-hdds/docs/content/design/ozone-enhancement-proposals.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Ozone Enhancement Proposals
+summary: Definition of the process to share new technical proposals with the Ozone community.
+date: 2019-06-07
+jira: HDDS-1659
+status: current
+author: Anu Enginner, Marton Elek
+---
+
+## Problem statement
+
+Some of the biggers features requires well defined plans before the implementation. Until now it was managed by uploading PDF design docs to selected JIRA. There are multiple problems with the current practice.
+
+ 1. There is no easy way to find existing up-to-date and outdated design docs.
+ 2. Design docs usually have better description of the problem that the user docs
+ 3. We need better tools to discuss the design docs in the development phase of the doc
+
+We propose to follow the same process what we have now, but instead of uploading a PDF to the JIRA, create a PR to merge the proposal document to the documentation project.
+
+## Non-goals
+
+ * Modify the existing workflow or approval process
+ * Migrate existing documents
+ * Make it harder to create design docs (it should be easy to support the creation of proposals for any kind of tasks)
+
+## Proposed solution
+
+ * Open a dedicated Jira (`HDDS-*` but with specific component)
+ * Use standard name prefix in the jira (easy to filter on the mailing list) `[OEP]
+ * Create a PR to merge the design doc (markdown) to `hadoop-hdds/docs/content/proposal` (will be part of the docs)
```

Comment:
During an offline discussion @arp7 had some concerns about the usability of reviewing markdown files in PR vs. reviewing google docs.

Reply:
Note: this is a complex PR because it contains the changes to show design docs on the docs page. A normal design doc would contain just one markdown file.

Reply:
Update: I removed the HTTP/template changes.